src/cljs/owlet_ui/routes.cljs

```diff
@@ -56,6 +59,9 @@
             (re/dispatch [:get-library-content-from-contentful params])
             (re/dispatch [:set-active-view :activity-view]))
 
+  (defroute "*" []
+            (set! (.-location js/window) "http://localhost:4000/#/404"))
```

Comment:
Hey @ZadenRB i know this is WIP but can we change this to a relative path ?
When we deploy the url will be different. 
`(set! (.-location js/window) "/#/404")`